utilities.js

```diff
@@ -0,0 +1,24 @@
+const products = require('./src/data/products')
+const categories = require('./src/data/categories')
+module.exports = {
+    getProductsByCategoryId : (categoryId) => {
+        return products.products.filter(value => value.categoryId === categoryId)
+    },
+    getCategoryById: (id) => {
+        return categories.categories.filter(value => value.id === id)[0];
+    },
+    getProductById: (id) => {
+        return products.products.filter(value => value.id === id)[0];
+    },
+    notFound: (res) => {
```

Comment:
as it's a utility method, better to start with a verb. i.e: `sendNotFoundResponse()`